src/components/orderItem.js

```diff
@@ -31,27 +54,32 @@ class OrderItem extends React.Component {
                 </div>
                 <div className="uk-width-1-2">
                     <div className="uk-grid">
-                        <div className="uk-width-1-3">
+                        <div className="uk-width-1-3"> 
                              {this.props.item.quantity_return_allowed > 0 &&
-                                <form>
-                                    <input type="number" id="returnQty" name="quantity" value="{this.props.item.quantity_return_allowed}"/>
-                                </form>
+                              <div>
+                                   <Options
+                                choices={elements}
+                                onChoice={(choice) => this.returnExchangeDetails(choice)} />
+                                <p>{this.props.item.validation_errors}</p>
+                            </div>
                               }
-                              {this.props.item.quantity_return_allowed === 0 &&
+                              {this.props.item.quantity_return_allowed == 0 &&
                                 <p>
-                                This item isn't returnable sorry
+                                    {!!(this.props.item.validation_errors) ? this.props.item.validation_errors : "" }
```

Comment:
Good for now. Will eventually need to "translate" these validation errors.
Validation Errors is an array, too, so the items will need to be looped over to display correctly.